tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -1532,3 +1518,28 @@ public function testZeroOffsetWithoutLimitIsIgnored() : void
         );
     }
 }
+
+interface GetCreateTableSqlDispatchEventListener
```

Comment:
I hope one day it gets fixed in the event dispatcher so that instead of just calling random methods dynamically, it lets one define an interface and accepts its implementors as listeners. Otherwise, it looks barely manageable.